### PR TITLE
Fix docs sidebar (do not auto display links for headers)

### LINKED ISSUE #195: Unintended behaviour of sidebar due to changes in 8ab8107
Since 8ab8107, the sidebar now treats h2 and h3 as subheadings, which makes navigation even more confusing.

Screenshot from 2024-07-13 16-35-06

### PR DESCRIPTION
**Pull request by marcsadler**
Closes #195

### CODE REVIEW

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ export default {
         link: 'https://forum.chrultrabook.com/'
       }
     ],
+	headerDepth: 0,
```
Comment (ethanaobrien) [inline src/.vuepress/config.js:51]: The spacing should match the rest of the file (4 spaces) and not be a tab lol

**Review by ethanaobrien — CHANGES_REQUESTED** [review]

File: src/.vuepress/config.js (hunk 1 above)
Comment (marcsadler) [reply src/.vuepress/config.js:51]: Gotcha no problem, but the whole of the file is using 2 spaces rn?

Comment (ethanaobrien) [reply src/.vuepress/config.js:51]: Sorry.. I just woke up and apparently can't count. 2 spaces then :)

Comment (marcsadler) [reply src/.vuepress/config.js]: No worries, I'll use a more reasonable text editor in future:)

**Review by ethanaobrien — APPROVED** [review]

**Review by WeirdTreeThing — APPROVED** [review]

**Pull request merged**